python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -258,21 +272,27 @@ def check_results(batch_df, batch_id):
                     [
                         Row(id="ttl-count-0", count=2),
                         Row(id="count-0", count=2),
+                        Row(id="ttl-list-state-count-0", count=3),
                         Row(id="ttl-count-1", count=2),
                         Row(id="count-1", count=2),
+                        Row(id="ttl-list-state-count-1", count=3),
                     ],
                 )
             elif batch_id == 2:
                 # ttl-count-0 expire and restart from count 0.
-                # ttl-count-1 get reset in batch 1 and keep the state
+                # ttl-count-1 get reset in batch 1 and keep the state.
```

Comment:
nit: it may not be very clear what "reset" means here - now I get what you mean, "TTL reset", but I was puzzled and had to think a bit.

Reply:
Updated the comment with more details, thanks!